components/tier.js

```diff
@@ -0,0 +1,92 @@
+
+function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Item from './item.jsx';
+
+var Tier = function (_React$Component) {
+  _inherits(Tier, _React$Component);
+
+  function Tier(props) {
+    _classCallCheck(this, Tier);
+
+    var _this = _possibleConstructorReturn(this, (Tier.__proto__ || Object.getPrototypeOf(Tier)).call(this, props));
+
+    _this.state = {
+      // Some of this will move to Item component
+      // But I want all of it together so I can easily see it all at once
+      bundleId: 1,
+      tier: {
+        tierId: 1, // from this service
+        tierCost: 1, // minimum cost
+        items: {
+          item1: {
+            itemId: 1, // from Tiers
+            itemName: 'game 1', // from Item service
+            itemImage: '', // from Item service
+            availability: '', // from System Requirements service
+            feedback: {
+              positiveReviews: 90, // from Reviews service
+              numberOfReviews: 526, // from Reviews service
+              reviewsFrom: 'Steam' // from Reviews service
+            }
+          },
+          item2: {
+            itemId: 2, // from this service
+            itemName: 'game 2', // from Item service
+            itemImage: '', // from Item service
+            availability: '', // from System Requirements service
+            feedback: {
+              positiveReviews: 76, // from Reviews service
+              numberOfReviews: 9652, // from Reviews service
+              reviewsFrom: 'Steam' // from Reviews service
+            }
+            // Will need to extend to max number of items later
+          } }
+      }
+    };
+    return _this;
+  }
+
+  _createClass(Tier, [{
+    key: 'getTierData',
+    value: function getTierData() {
+      var _this2 = this;
+
+      var tierId = this.state.tier.tierId;
+      $.ajax({
+        method: 'GET',
+        url: '/itemsIncluded',
+        // url: './itemsIncluded:' + tierId,
+        datatype: 'json'
+      }).done(function (data) {
+        _this2.setState({
+          //TODO
+        });
+      });
+    }
+  }, {
+    key: 'render',
+    value: function render() {
+      return React.createElement(
+        'div',
+        null,
+        React.createElement(
+          'div',
+          { className: 'item' },
+          React.createElement(Item, null)
+        )
+      );
+    }
+  }]);
+
+  return Tier;
+}(React.Component);
+
+export default Tier;
```

Comment:
Note the red circle with a dash.  This visual cue indicates you are missing a blank line at the bottom of your file.  That's another standard for writing clean code that I want you to adopt.

Good news...  you can modify your editor settings so that it automatically does a number of housekeeping tasks to manage code quality.  They include, removing trailing whitespace from lines and ensuring there's a blank line at the ends of your files.  Set it in your preferences and you never have to worry about it.